codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/ClassSweepCompilerPluginComponent.scala

```diff
@@ -15,27 +15,40 @@ class ClassSweepCompilerPluginComponent(options: CodecRegistrationCheckerOptions
   override val runsAfter: List[String] = List("refchecks")
   override def description: String = s"searches for direct descendants of classes annotated with serializability trait"
 
-  val foundTypes: mutable.Buffer[(String, String)] = mutable.ListBuffer()
-  val typesToUpdate: mutable.Buffer[(String, String)] = mutable.ListBuffer()
+  val foundParentChildFullyQualifiedClassNamePairs: mutable.Buffer[ParentChildFullyQualifiedClassNamePair] =
```

Comment:
ok, let me change it in all occurences then

Reply:
done - only two values that do not use this abbreviation are here:

https://github.com/VirtusLab/akka-serialization-helper/blob/fix_codec_registration_checker_false_positives/codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala#L143

https://github.com/VirtusLab/akka-serialization-helper/blob/fix_codec_registration_checker_false_positives/codec-registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala#L145

I left these two because in my opinion in this cases using shortcuts like `fQCNsFromFoundTypes` would not look good and would not be readible